apps/ledger/libs/helpers/w3c.schema.builder.spec.ts

```
@@ -0,0 +1,442 @@
+    it('should create valid W3C schema with minimal attributes', () => {
+      const attributes = [
+        {
+          attributeName: 'name',
+          displayName: 'Full Name',
+          schemaDataType: W3CSchemaDataType.STRING,
+          isRequired: true,
+          description: 'The person full name'
+        }
+      ];
+
+      const result = w3cSchemaBuilder(attributes, 'Person Schema', 'A schema for person credentials') as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)